.github/workflows/test.yml

```diff
@@ -9,12 +9,21 @@ on:
       - v*
   workflow_dispatch:
 
+env:
+  AERIE_USERNAME: "${{secrets.AERIE_USERNAME}}"
+  AERIE_PASSWORD: "${{secrets.AERIE_PASSWORD}}"
+  HASURA_GRAPHQL_ADMIN_SECRET: "${{secrets.HASURA_GRAPHQL_ADMIN_SECRET}}"
+  HASURA_GRAPHQL_JWT_SECRET: "${{secrets.HASURA_GRAPHQL_JWT_SECRET}}"
+  POSTGRES_USER: "${{secrets.POSTGRES_USER}}"
+  POSTGRES_PASSWORD: "${{secrets.POSTGRES_PASSWORD}}"
```

Comment:
Since we're just using these as creds in a CI instance and these aren't connecting to any outside services, let's just hardcode vales for these environment variables instead of referencing GH environment secrets.

Reply:
Why? Isn't the environment already set up?

Reply:
It is, but is there any value added by using it? It seems unnecessarily complicated to me, but maybe there's something I'm missing.

Reply:
Noting and dropping this topic for this PR.